eng/mgmt/automation/sdk_generate.py

```diff
@@ -99,16 +99,16 @@ def sdk_automation(input_file: str, output_file: str):
     packages = []
     readme = config['relatedReadmeMdFile']
     match = re.search(
-        'specification/([^/]+)/resource-manager/readme.md',
+        '(specification)?/?([^/]+)/resource-manager/readme.md',
```

Comment:
What scenarios will we not have a specification folder in the path?

Reply:
Hi wes, the script will be used for shift-left, sdk automation pipeline, and so on. In azure-rest-api-specs repo, the swagger struct is repoRoot/specification/<service>/, but in others are not. for example, in shift-left, the service repo struct is repoRoot/src/swagger***/<service>.

Reply:
Will this structure even work in cases outside of the specs repo? What I mean by that is should we instead be passing in the list of files or the exact path instead of making assumptions about the layout in all our various scripts in each of the repos? I'd like us to stop hard-coding these assumptions if possible.